testgrid/specs/storage-migration.yaml

```diff
@@ -252,9 +248,7 @@
       localPVStorageClassName: local
   postInstallScript: |
     source /opt/kurl-testgrid/testhelpers.sh
-    create_deployment_with_mounted_volume "migration-test" "default" "/data" "registry:2.8.1"
     create_random_file_and_upload_to_deployment "migration-test" "default" "./test.data" "/data/test.data"
-    test_push_image_to_registry
     install_and_customize_kurl_integration_test_application
   postUpgradeScript: |
     source /opt/kurl-testgrid/testhelpers.sh
```

Comment:
you need to remove test_pull_image from this

Reply:
Is not it done already? Are we removing ` test_push_image_to_registry` should we remove any other test/check done in this scenario?

Reply:
it's currently 4 lines below this comment, on line 258. (why github won't let you comment on lines outside the fold I will never understand)

If there's no registry, you can't pull from the registry.

Reply:
done